src/judge.rs

```diff
@@ -53,11 +53,6 @@ impl Judge {
         None
     }
 
-    #[cfg(test)]
-    pub fn short_dict() -> Self {
-        Self::new(vec!["BIG", "FAT", "JOLLY", "AND", "SILLY", "FOLK", "ARTS"]) // TODO: Collins 2018 list
-    }
-
```

Comment:
Moved test func to the test module to clean it up (imo)

Reply:
(It does mean this is duplicated into the `moves.rs` test module as well, but I much prefer tests to be self contained in this way so I think that's a net +)

Reply:
Ah fair enough. What do you think of importing test modules from other test modules? I import `trivial_bag` from `bag::tests` to `hand::tests` and `moves::tests` because I find it keeps the tests about the question at hand.

Reply:
Don't know if I have a strong feeling there. I'd maybe make a top-crate-level test mod that has generic utils? Just to avoid changes to one modules tests breaking others